app/navigation/nonsipp/SharesNavigator.scala

```diff
@@ -70,6 +71,9 @@ object SharesNavigator extends JourneyNavigator {
     case SharesIndividualSellerNINumberPage(srn, index) =>
       controllers.routes.UnauthorisedController.onPageLoad()
 
+    case CompanyNameOfSharesSellerPage(srn, index) =>
```

Comment:
Should navigate to CRN page here, but this is fine to do later on under you next ticket @narendrapaduchuri

Reply:
Hi @kberec , will fix it in my upcoming ticket